sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -2255,15 +2255,16 @@ matrix(
             },
           });
           assert.equal(serialized, expectedSerialized);
-          assert.equal(poller.operationState.status, "succeeded");
+          assert.isNotNull(poller.operationState);
+          assert.equal(poller.operationState!.status, "succeeded");
```

Comment:
I see the issue now.  `assert.isUndefined()` above makes the compiler to infer the type as `undefined`.  And the later assertion won't help any more when operating on `undefined` type.

I think `assert.exists()` is still better? as `isNotNull` asserts that the value is not null.

```suggestion
          assert.exists(poller.operationState);
          assert.equal(poller.operationState.status, "succeeded");
```